verilog/formatting/formatter_test.cc

```diff
@@ -597,7 +597,10 @@ static const std::initializer_list<FormatterTestCase> kFormatterTestCases = {
     },
     {
         "  parameter  int   foo=$bar(-  z,- y ) ;",
-        "parameter int foo = $bar(-z, -y);\n",
+        "parameter int foo = $bar(\n"
```

Comment:
Yes, It was because of `kAlwaysExapand` partition policy applied to function calls arguments section.
Generally I tried to focus on function calls arguments rather. The simplest way to accomplish that was to
skip creating own section for function calls (with an exception for standalone calls) and just partition function arguments.
Which gives for example such partition tree:
```
  { ([<auto>], policy: fit-else-expand) @{49}, (origin: "parameter  ...  z,- y ) ;")
    { ([parameter int foo = $bar (], policy: fit-else-expand) }
    { (>>>>[<auto>], policy: always-expand) @{49,1}, (origin: "-  z,- y")
      { (>>>>[- z ,], policy: fit-else-expand, (origin: "-  z")) }
      { (>>>>[- y], policy: fit-else-expand, (origin: "- y")) }
    }
    { ([) ;], policy: uninitialized) }
  }
```
I thought that it'd be better to always expand such arguments list rather than fit some of them and left dangling `);` at the end, like in formatter tuning test:
```
module m;
  assign wwwwww[77:66] = sss(
      qqqq[33:22], vv[44:1]
  );
endmodule
```
(I left dangling `);` because I thought that it'd be better to address function calls in trailing assignments in separate PR)